lnrpc/rpc.proto

```diff
@@ -3323,6 +3330,18 @@ message BakeMacaroonResponse {
     string macaroon = 1;
 }
 
+message MacaroonPermissionList {
+    // A list of macaroon permissions.
+    repeated MacaroonPermission permissions = 1;
+}
+
+message ListPermissionsRequest {
+}
+message ListPermissionsResponse {
+    // The hex encoded macaroon, serialized in binary format.
```

Comment:
The description seems to be not correct.

Reply:
Right, thanks!